core/src/test/java/org/infinispan/test/AbstractInfinispanTest.java

```diff
@@ -91,16 +91,16 @@ protected void eventuallyEquals(int expected, IntSupplier supplier) {
             () -> expected == supplier.getAsInt());
    }
 
-   protected void eventuallyEquals(String message, long expected, LongSupplier supplier) {
+   public static void eventuallyEquals(String message, long expected, LongSupplier supplier) {
```

Comment:
You can extend `MultipleCacheManagersTest` in both tests, there's no rule that non-clustered tests have to use `SingleCacheManagerTest`.